bridges/TelegramBridge.php

```diff
@@ -43,7 +43,10 @@ public function collectData() {
 			$item['title'] = html_entity_decode($this->itemTitle, ENT_QUOTES);
 			$item['timestamp'] = $this->processDate($messageDiv);
 			$item['enclosures'] = $this->enclosures;
-			$author = trim($messageDiv->find('a.tgme_widget_message_owner_name', 0)->plaintext);
+			$author = trim($messageDiv->find('span.tgme_widget_message_from_author', 0)->plaintext);
+			if (empty($author)) {
+				$author = trim($messageDiv->find('a.tgme_widget_message_owner_name', 0)->plaintext);
+			}
```

Comment:
Instead of getting the 'from_author' first, get 'owner_name' (as its always returned) and then check for 'from_author'.
```PHP
	$author = trim($messageDiv->find('a.tgme_widget_message_owner_name', 0)->plaintext);

	if ($messageDiv->find('span.tgme_widget_message_from_author', 0)) { // Use 'from author' if given
		$author = trim($messageDiv->find('span.tgme_widget_message_from_author', 0)->plaintext);
	}